docs/specviz/import_data.rst

```diff
@@ -26,11 +26,12 @@ and choose a file that can be parsed as a :class:`~specutils.Spectrum1D` in the
 
 .. image:: img/import_data_2.png
 
-After clicking :guilabel:`Import`, the data file will be parsed and loaded into the application. A notification will appear to let users know if the data import was successful. Afterward, the new data set can be found in the :guilabel:`Data` tab of each viewer's menu.
-
-.. image:: img/import_data_3.png
-
-Here, users can select the loaded data set to be visualized in the viewer.
+After clicking :guilabel:`Import`, the data file will be parsed and loaded into the
+application. A notification will appear to let users know if the data import was 
+successful, and if possible the data will be immediately displayed (this may not 
+be possible if an incompatible spectrum was already displayed). Afterward, the new 
```

Comment:
Maybe clarify what one should expect when incompatible spectrum is displayed? Will it crash at loading/selection or something?